src/main/java/com/jfrog/xray/client/impl/services/summary/VulnerableComponentsImpl.java

```diff
@@ -0,0 +1,22 @@
+package com.jfrog.xray.client.impl.services.summary;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.jfrog.xray.client.services.summary.VulnerableComponents;
+
+import java.util.List;
+
+/**
+ * @author yahavi
+ */
+public class VulnerableComponentsImpl implements VulnerableComponents {
+
+    @SuppressWarnings("unused")
+    @JsonProperty("fixed_versions")
+    private List<String> fixedVersions;
```

Comment:
I think that *fixedVersions* should eb changed to *fixVersions*.
We shgould change this throughout the new code.

Reply:
We agreed verbally to leave it "Fixed versions".